test-requirements.txt

```diff
@@ -11,6 +11,7 @@ mock==1.0.1
 nose2==0.13.0
 pbr==3.1.1
 python-mimeparse==1.6.0
-testtools==2.3.0
+testtools==2.3.0; python_version<"3"
+testtools==2.5.0; python_version>="3"
```

Comment:
Is there a reason to pin these this strictly? I realize you didn't change this but an explicit PATCH version seems like you can't get bug fixes, etc.